src/state/queries/post-feed.ts

```diff
@@ -143,13 +89,23 @@ export function usePostFeedQuery(
     queryKey: RQKEY(feedDesc, params),
     async queryFn({pageParam}: {pageParam: RQPageParam}) {
       logger.debug('usePostFeedQuery', {feedDesc, pageParam})
-      if (!pageParam) {
-        tuner.reset()
-      }
-      const res = await api.fetch({cursor: pageParam, limit: 30})
+
+      const {api, tuner, cursor} = pageParam
+        ? pageParam
```

Comment:
The `pageParam` is attached to the RQ cache. Each page has its own page param, created from the previous page in `getNextPageParam()`. Essentially what this is doing is threading the `api` and `tuner` through all of those `pageParams`, to look like this:

```
page1: {api, tuner, cursor: cursor1}
page2: {api, tuner, cursor: cursor2}
page3: {api, tuner, cursor: cursor3}
// ...        ^ same^     ^changes
```

With each fetch, we're only given the `pageParam` for the upcoming page, so we gotta thread `api` and `tuner` through all of them.

And -- big picture -- we thread that data through the page params so that their lifecycle stays 1:1 with the RQ cache lifecycle.